gst/rtsp-server/rtsp-client.c

```diff
@@ -101,7 +101,7 @@ static GHashTable *tunnels;     /* protected by tunnels_lock */
 /* FIXME make this configurable. We don't want to do this yet because it will
  * be superceeded by a cache object later */
 #define WATCH_BACKLOG_SIZE              100
-#define WATCH_BACKLOG_BYTES             10000000
+#define WATCH_BACKLOG_BYTES             5000000
```

Comment:
Perhaps an explanation of why this was reduced / why this size?

Reply:
This is a cherry-pick of an old commit. Refactoring old code is not in the remit of this PR.